src/commands/scaling/addScaleRule/queue/QueueAuthSecretStep.ts

```diff
@@ -0,0 +1,29 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { Secret } from '@azure/arm-appcontainers';
+import { AzureWizardPromptStep, nonNullProp } from '@microsoft/vscode-azext-utils';
+import { QuickPickItem } from 'vscode';
+import { localize } from '../../../../utils/localize';
+import { IAddScaleRuleWizardContext } from '../IAddScaleRuleWizardContext';
+
+export class QueueAuthSecretStep extends AzureWizardPromptStep<IAddScaleRuleWizardContext> {
+    public async prompt(context: IAddScaleRuleWizardContext): Promise<void> {
+        const noSecrets: string = localize('noSecretsFound', 'No secrets were found. Create a secret to proceed.');
```

Comment:
nit: I personally prefer to initializing variables closer to where they are being used.